test/integration/TokensTest.php

```diff
@@ -52,30 +53,6 @@ function testObtainNewToken(){
         $this->assertContains($this->clientId, $scopes);
     }
 
-    /**
-     * Retrieving a token with the same credentials should be cached
-     * @return null
-     */
-    public function testObtainCachedToken()
```

Comment:
Moved this test to the unit tests to introduce some mocking as tokens with an expiry time less than 60 seconds are not cached (dev environment uses 20 second tokens)